src/components/sync-connected-sites.tsx

```diff
@@ -188,7 +182,10 @@ const SyncConnectedSitesSection = ( {
 					const hasPullFinished = sitePullState && isKeyFinished( sitePullState.status.key );
 
 					const pushState = getPushState( selectedSite.id, connectedSite.id );
-					const isPushError = pushState.isError;
+					const isPushing = pushState && isKeyPushing( pushState.status.key );
+					const isPushError = pushState && isKeyFailed( pushState.status.key );
+					const hasPushFinished = pushState && isKeyFinished( pushState.status.key );
+
```

Comment:
This change ensures that `getPullState` and `getPushState` can have the same return type.